python/examples/integrations/Fugue_Profiling.ipynb

```diff
@@ -1071,7 +1071,7 @@
     }
    ],
    "source": [
-    "!ls /tmp/output*.parquet"
+    "# !ls /tmp/output*.parquet"
```

Comment:
this is the command that made my changes fail. i have no clue why and I couldn't repro this locally. on my branch the tests pass. 
apparently it's something that only happens on Ubuntu environments? since Anthony could narrow it down to this line of code. 
i feel awkward to make this change, but can't find a way to make this work without it.